miniapps/nurbs/nurbs_ex24.cpp

```diff
@@ -0,0 +1,458 @@
+//                                MFEM Example 24 -- modified for NURBS FE
+//
+// Compile with: make nurbs_ex24
+//
+// Sample runs:  nurbs_ex24 -m ../../data/pipe-nurbs-2d.mesh
```

Comment:
The first sample run fails.

Reply:
H-div and H-curl NURBS extensions for now only work for a single patch.
The logic for deciding which knotvectors relate to which direction has not been implemented yet.

- I'll remove the multipatch examples.
- Add a check to the examples.
- Add restriction in the comments.

Reply:
Thanks for catching this!

Reply:
The patch check is already present in the NURBS extension in `GetDivExtension` and  `GetCurlExtension`.

Reply:
The first example does work when `-o 2` is added. The mesh is quadratic.